pagopa/io-app
# chore(IT Wallet): [SIW-1862] Remove transparent header within credential details screen

## Short description
This PR removes the header transparency within credential details screen

## List of changes proposed in this pull request
- Removed `transparent: true` prop from `useHeaderSecondLevel`
- Removed header images for TS credential
- Removed extra top margins from details screens

## How to test
Navigate to the credential details screen, check that everything is rendered correctly.

## Preview
<img src="https://github.com/user-attachments/assets/fa9b5564-478f-40ee-bc06-327862d8716f" width="200" />
